src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length <= 1) {
+    return {};
+  }
+
+  const elements = sourceString.trim().split(';');
```

Comment:
Consider chaining the `trim` and `split` methods to improve readability. Make sure that all chained methods are indented at the same level.